libs/shared/error/src/lib/error.ts

```diff
@@ -25,3 +25,5 @@ export class TypeError extends BaseError {
     super(...args);
   }
 }
+
+export const GENERIC_ERROR_MESSAGE = 'Something went wrong';
```

Comment:
This `Something went wrong` error message is specific to `payments-next`, so it shouldn't be in shared library.